package.json

```diff
@@ -26,6 +26,7 @@
     "@types/node": "14.14.25",
     "@typescript-eslint/eslint-plugin": "^4.14.2",
     "@typescript-eslint/parser": "^4.14.2",
+    "cz-conventional-changelog": "^3.3.0",
```

Comment:
Not sure if this is included intentionally, but it might be better to decouple adding this dependency from implementing the `GuSecret` construct?